.github/workflows/backend-ort.yml

```diff
@@ -0,0 +1,41 @@
+name: backend-ort
+
+on:
+  push:
+    paths-ignore:
+      - "demo/**"
+      - "tools/**"
+
+  pull_request:
+    paths-ignore:
+      - "demo/**"
+      - "tools/**"
+      - "docs/**"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  script_install:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.7]
+        torch: [1.9.0]
+        include:
+          - torch: 1.9.0
+            torch_version: torch1.9
+            torchvision: 0.10.0
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install mmdeploy
+        run: |
+          python3 tools/scripts/build_ubuntu_x64_ort.py
```

Comment:
could we add the simple test to check mmdeploy+ncnn is installed correctly? 
maybe `python -c 'import mmdeploy.apis.ncnn as ncnn_api; assert ncnn_api.is_available() and ncnn_api.is_custom_ops_available()'`
and run `python tools/check_env.py` and print the env info.

Reply:
may verify `python -c 'import mmdeploy.apis.onnxruntime as api; assert api.is_available() and api.is_custom_ops_available()'`